static/less/cases.less

```diff
@@ -62,14 +74,34 @@ footer {
   }
 }
 
+.glyphicon-duplicate:before {
+  content: "\e224";
+}
+
+tr + .emptymessage {
+    display: none;
+}
+.table-striped > tbody > tr.emptymessage > td{
+  background-color: #fff;
+}
+
+::-ms-clear {
+  display: none;
+}
+
+.input-clear .glyphicon {
+      pointer-events: all;
+    }
+
 /**
  * Smartmin tweaks - consistent layout for read page tables
  */
 .read-label {
-  width: 33%
+  width: 10%
```

Comment:
These tweaks are just for the FAQ read page right? If so then better to put them at the bottom of read template